configs/defaults/gcnv.toml

```diff
@@ -4,9 +4,14 @@ intervals_path = 'gs://cpg-common-main/references/hg38/v0/exome_calling_regions.
 exclude_intervals = ['chrM']
 # Aim for 10-50Mbp genomic coverage per shard
 interval_shards = [['chr1'], ['chr2', 'chr3'], ['chr4', 'chr5'], ['chr6', 'chr7'], ['chr8', 'chr9', 'chr10'], ['chr11', 'chr12'], ['chr13', 'chr14', 'chr15'], ['chr16', 'chr17'], ['chr18', 'chr19', 'chr20'], ['chr21', 'chr22', 'chrX', 'chrY']]
-ploidy_priors = 'gs://cpg-common-main/references/gatk-sv/hg38/v0/sv-resources/resources/v1/hg38.contig_ploidy_priors_homo_sapiens.tsv'
+#ploidy_priors = 'gs://cpg-common-main/references/gatk-sv/hg38/v0/sv-resources/resources/v1/hg38.contig_ploidy_priors_homo_sapiens.tsv'
```

Comment:
Can we just rm this if we don't need it anymore?

Reply:
Yup! There were two files added to this WF's config - one is interval regions which should be relative to this exact cohort's exome capture, but this looks value was the same as the default in the analysis-runner config so I've removed from this WF-specific config